tests/CMakeLists.txt

```diff
@@ -118,30 +119,37 @@ set(func_binaries
 
 set(comp_binaries
     "simple"
+    "simple_libc"
+    "simple_call_internal"
+    "simple_call_external"
+    "simple_external"
     #"time"
     #"lua_simple"
     #"lua_script"
-    "args_simple"
+    #"args_simple"
     #"test_two_comps-comp1"
     #"test_two_comps-comp2 0x2000000"
     )
 
 set(tests
     "simple"
+    "simple_libc"
+    "simple_call_internal"
+    "simple_call_external"
     #"time"
     #"lua_simple"
     #"lua_script"
     "test_map"
     #"test_args_near_unmapped"
     #"test_two_comps"
     #"test_two_comps_inter_call"
-    "args-simple args_simple check_simple 40 2"
-    "args-more args_simple check_simple 40 2 2 2" # Check additional arguments are ignored
-    "args-combined args_simple check_combined 400 2 20"
-    "args-negative args_simple check_negative -42"
-    "args-long-max args_simple check_llong_max 9223372036854775807"
-    "args-long-min args_simple check_llong_min -9223372036854775808"
-    "args-ulong-max args_simple check_ullong_max 18446744073709551615"
+    #"args-simple args_simple check_simple 40 2"
```

Comment:
Any reason for commenting these tests out in this PR?

Reply:
Reason is PEBCAK. Re-added in a01fe1e2eb64bb491ae4cb6e13f7c0742d53b303.